__tests__/app.test.js

```diff
@@ -76,3 +78,37 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: Check the returned results keys are correct", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        res.body.articles.rows.forEach((article) => {
```

Comment:
this test will pass if 0 articles are returned as if there are 0 elements in the array, the callback fn inside the forEach will not be invoked and the test will pass by default. you'll need to check that the array length is what you expect it to be first, and that way you know the array is not empty